context/context.go

```diff
@@ -57,6 +56,7 @@ type Context struct {
 // AnkhConfig defines the shape of the ~/.ankh/config file used for global
 // configuration options
 type AnkhConfig struct {
+	Include                   []string           `yaml:"include"`
```

Comment:
Shouldn't this be a private field? IIUC it's filled in by code and not parsed from the yaml.